spec/spec_helper.rb

```diff
@@ -2,7 +2,7 @@
 require File.expand_path("../../config/environment", __FILE__)
```

Comment:
After you update to rspec 3, backup this file and regenerate via `rails g rspec:install`. RSpec 3 adds in spec_helper *and* rails_helper.

Remove the giant block comment from spec_helper. It has sensible defaults (mostly) such as enforcing the new syntax (which transpec can convert all the files to)

Move anything extra that was added in this spec file to its own file in spec/support. e.g. Since we're using capybara, create spec/support/capybara.rb and include pertinent code there. I like to keep the generated helper files at their defaults as it makes understanding what we've changed easier (it's only the stuff in spec/support).

You'll have to uncomment the line in rails_helper that includes all the files in spec/support